### ISSUE TITLE
cosmetic issues in in-progress tasks returned payload

### ISSUE THREAD

**Issue by amolari** [F5Networks/f5-appsvcs-extension#828]
### Environment
 * Application Services Version: 3.50.2
 * BIG-IP Version: not relevant

### Summary
Posting an async declaration the following payload is returned:
```
{
  "id": "dc5c5f6d-1e86-4428-b643-e6da3a9b72ed",
  "results": [
    {
      "message": "Declaration successfully submitted",
      "tenant": "",
      "host": "",
      "runTime": 0,
      "code": 0,
      "declarationId": "691121"
    }
  ],
  "declaration": {},
  "selfLink": "https://localhost/mgmt/shared/appsvcs/task/dc5c5f6d-1e86-4428-b643-e6da3a9b72ed"
}

```

when querying the task-id, if the task is in-progress we get the following payload returned:
```
{
  "id": "dc5c5f6d-1e86-4428-b643-e6da3a9b72ed",
  "results": [
    {
      "message": "in progress",
      "tenant": "",
      "host": "",
      "runtTime": 0,
      "code": 0
    }
  ],
  "declaration": {}
}
```

Issues:

1.  typo in `runtTime` => should be `runTime` (code: https://github.com/F5Networks/f5-appsvcs-extension/blob/91e3b2dd2be4369b95eaea039899bd2174f48830/src/lib/asyncHandler.js#L98)
2. `declarationId` is missing
3. `selfLink` is missing

**Comment by mdditt2000** [comment]
@amolari thanks for opening this cosmetic issue. Easy fix on your end

--- Change "runtTime": 0, to runTime

Added AUTOTOOL-4353

**Comment by mdditt2000** [comment]
@amolari picked up in the current sprint

**Comment by sunitharonan** [comment]
This issue has been resolved in AS3 52.0
